app/src/main/java/com/everest/userinfo/DisplayActivity.kt

```diff
@@ -0,0 +1,25 @@
+package com.everest.userinfo
+
+import android.annotation.SuppressLint
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.everest.userinfo.databinding.AcitvityDisplayBinding
+import com.everest.userinfo.databinding.ActivityHomeBinding
+
+class DisplayActivity : AppCompatActivity() {
+    private lateinit var binding : AcitvityDisplayBinding
+    @SuppressLint("SetTextI18n")
+    override fun onCreate(savedInstanceState: Bundle?) {
+
+        super.onCreate(savedInstanceState)
+        binding = AcitvityDisplayBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        val bundle = intent.extras
+        binding.resultTV.text=
+            "Hi "+ (bundle?.getString("userName")) +", How are you? Are you staying at "+bundle?.getString("userAddress")+"-"+
```

Comment:
We should use strings from resources. We have a concept called string templates and formatting
https://stackoverflow.com/questions/3656371/is-it-possible-to-have-placeholders-in-strings-xml-for-runtime-values